src/game/route.py

```diff
@@ -50,6 +51,38 @@ def find_game_player(cur_game_players: GamePlayers, player_id: str) -> Optional[
                  if game_player.player_venmo_username == player_id), None)
 
 
+PaymentURL = NamedTuple('PaymentURL', [('url', str), ('is_send', bool)])
```

Comment:
We should add some short docs on this, esp what `is_send` is suppose to signify.